sql/derived_tables/po_lines_eresource.sql

```diff
@@ -18,12 +17,22 @@ WITH temp_eresource AS (
         json_extract_path_text(data, 'eresource', 'userLimit') AS pol_user_limit,
         json_extract_path_text(data, 'eresource', 'resourceUrl') AS pol_resource_url
     FROM
-        po_lines AS pol
+        po_lines AS pol WHERE json_extract_path(data, 'eresource') IS NOT NULL
 )
 SELECT
     te.pol_id,
-    te.pol_location_id,
-    il.name AS location_name,
+    json_extract_path_text(locations.data, 'holdingId') AS pol_holding_id,
+    ih.hrid AS pol_holding_hrid,
+    CASE WHEN json_extract_path_text(locations.data, 'locationId') IS NOT NULL
+	    THEN json_extract_path_text(locations.data, 'locationId') 
+		ELSE ih.permanent_location_id END AS pol_location_id,
+	CASE WHEN (il.name) IS NOT NULL THEN il.name
```

Comment:
Attribute is surrounded by brackets. The condition should be on a single line and the case distinction on an extra line.

Reply:
Have changed that.